assets/js/apps/search/components/ResultList.stories.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import { PureResultList } from './ResultSection'
+import { action } from '@storybook/addon-actions';
+
+export default {
+    component: PureResultList,
+    title: 'Result list',
+    decorators: [story => <div style={{ padding: '3rem'}}>{story()}</div>],
+    excludeStories: /.*Data$/
+};
+
+export const allAccessResultData = {
+    id: "1",
+    type: "experiment",
+    title: "siNET",
+    url: "",
+    size:"35MB",
```

Comment:
Codacy has a fix for the issue: [Missing space before value for key 'size'.](https://app.codacy.com/gh/mytardis/mytardis/pullRequest?prid=7933948)
```suggestion
    size: "35MB",
```